game/game.py

```diff
@@ -387,16 +387,17 @@ def initialize_turn(
             for_red: True if opfor should be re-initialized.
             for_blue: True if the player coalition should be re-initialized.
         """
+        # Check for win or loss condition FIRST!
```

Comment:
If we don't check for the win/loss condition first, we'll get an exception in self.set_bullseye() which should be fairly obvious since bullseye is set on the closest enemy CP resepectively and in case of a win/loss one of the the teams has no CPs left. It also speaks for itself that there's no need to set the bullseyes anymore after a campaign is done...